rust/projects/file-io/Cargo.toml

```diff
@@ -7,4 +7,6 @@ edition = "2018"
 
 [dependencies]
 clap = "2.32.0"
-failure = "0.1.5"
\ No newline at end of file
+failure = "0.1.5"
+serde = { version = "1.0.89", features = ["derive"] }
```

Comment:
I did not know you could get the derive macros directly from the serde crate. Neat!